### PR TITLE
Sprint12 05 a46724 reduce ntp logging verbosity

### PR DESCRIPTION
**Pull request by arangamani**
Changed the template ntp.conf.erb to include only major events in log. It
 was checked by running the server for few hours and by changing the time
 manually which wrote a single line in the /var/log/messages.
 {{{
 Jun  7 14:07:03 ip-10-36-10-212 ntpd[3831]: time reset -610.032977 s
 }}}

### CODE REVIEW

File: cookbooks/sys_ntp/templates/default/ntp.conf.erb [hunk 1]
```diff
@@ -13,6 +13,7 @@ tinker panic 0 dispersion 1.000
 
 driftfile /var/lib/ntp/ntp.drift
 statsdir /var/log/ntpstats/
+logconfig =syncstatus -syncall
```
Comment (ekargard) [inline cookbooks/sys_ntp/templates/default/ntp.conf.erb:16]: Looks good.  That should quite logging down.

**Comment by ekargard** [comment]
Go ahead and merge

**Comment by arangamani** [comment]
Is it a command to me?

On Thu, Jun 7, 2012 at 6:31 PM, ekargard <
reply@reply.github.com

**Comment by ekargard** [comment]
Command??  More like a request.  There may be multiple reviewers so each of one needs to give the go ahead to merge.  Once everyone agrees the developer can do the merge.  In changes like this one I think the reviewer should just go ahead and do the merge.

**Comment by arangamani** [comment]
I didn't mean to stress on that word 'command'. I wanted to make sure if
you asked me to do it or someone else to. I'll go ahead and merge it.

On Thu, Jun 7, 2012 at 8:03 PM, ekargard <
reply@reply.github.com

**Pull request merged**